src/test/unit/controller/ChecklistController.test.ts

```diff
@@ -0,0 +1,21 @@
+import sinon from 'sinon';
+import { Response } from 'express';
+import ChecklistController from '../../../main/controllers/ChecklistController';
+import { mockRequest } from '../mocks/mockRequest';
+
+const checklistController = new ChecklistController();
+
+describe('Checklist Controller', () => {
+  const t = {
+    checklist: {},
+  };
+
+  it('should render the checklist page', () => {
+    const response = { render: () => '' } as unknown as Response;
```

Comment:
Please use mockResponse() object instead of redefining your own.

Reply:
Change applied - build successful